.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -7,18 +7,27 @@ assignees: ''
 
 ---
 
-**Release**: [v0.x.x](https://github.com/GenericMappingTools/pygmt/milestones/0.x.x)
+**Release**: [v0.x.x](https://github.com/GenericMappingTools/pygmt/milestones/?)
 **Scheduled Date**: YYYY/MM/DD
+**Pull request due date**: YYYY/MM/DD
```

Comment:
Need blank lines in between, otherwise, they will show up in one line.
```suggestion
**Release**: [v0.x.x](https://github.com/GenericMappingTools/pygmt/milestones/?)

**Scheduled Date**: YYYY/MM/DD

**Pull request due date**: YYYY/MM/DD
```

Reply:
I tried creating a release issue at https://github.com/GenericMappingTools/pygmt/issues/new?assignees=&labels=maintenance&template=release_checklist.md&title=Release+PyGMT+x.x.x and don't think this shows up on one line. Probably don't need it?

Reply:
Yes.